pkgs/os-specific/linux/prl-tools/default.nix

```diff
@@ -39,7 +56,10 @@ stdenv.mkDerivation rec {
     fi
   '';
 
-  patches = lib.optionals (lib.versionAtLeast kernel.version "5.18") [ ./prl-tools.patch ];
+  patches = lib.concatMap
```

Comment:
This is pretty short written out so I would prefer that. Also it makes it easier to add more patches later on.

Reply:
Done.